### PR TITLE
🎉 Create eds-utils

### LINKED ISSUE #1846: Create eds-utils package
Create a new package `eds-utils` in `packages` for sharing utility and hooks between `eds-core-react` & `eds-lab`.

* Copy `utils` and `hooks` from `eds-core-react`

### PR DESCRIPTION
**Pull request by oddvernes**
resolves #1846

### CODE REVIEW

**Review by mimarz — CHANGES_REQUESTED** [review]
Looking good, some small nitpicking.

* You can removed `nginx.conf` as there is nothing here we will host on a webserver standalone.
* Suggest add a `build:utils` and `test:utils` scripts to the root `package.yaml` so we have the same shortcuts as for our other packages.

**Comment by oddvernes** [comment]
test fails btw because there are no tests. So we should either not add test as ci step, or add a test/mock test

**Comment by mimarz** [comment]
> test fails btw because there are no tests. So we should either not add test as ci step, or add a test/mock test

Just make a simple test for for `useMounted` perhaps?

File: packages/eds-utils/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# @equinor/eds-utils
+
+This package host hooks and utility functions that are used by eds-core-react and eds-lab.
```
Comment (vnys) [inline packages/eds-utils/README.md:3]:
```suggestion
This package host hooks and utility functions that are used by [eds-core-react](https://www.npmjs.com/package/@equinor/eds-core-react) and [eds-lab](https://www.npmjs.com/package/@equinor/eds-lab).
```

Comment (vnys) [reply packages/eds-utils/README.md:3]: Add links to eds-core-react and eds-lab

**Review by vnys — CHANGES_REQUESTED** [review]
Just some missing info. Also, we should add some documentation to the Storybook. Have a look at Circuit UI’s storybook for an example.

**Comment by vnys** [comment]
About the tests… I think we need to agree on _why_ we write tests. I don’t see the value of writing a test just to make some system shut up. If we have 10 modules, and just one test, what’s the point? If we only focus on moving issues to the right on the board, does that mean that the job is done if we cut corners to get there? Or does it mean we have accumulated even more technical debt? If the point is to publish the package as quickly as possible because it’s a blocker, then we should at least create an issue for writing tests and move it to the top of the todo lane. We need to uphold a certain level of quality in EDS, having a good test coverage and writing good documentation is part of that.

**Comment by mimarz** [comment]
> About the tests… I think we need to agree on _why_ we write tests. I don’t see the value of writing a test just to make some system shut up. If we have 10 modules, and just one test, what’s the point? If we only focus on moving issues to the right on the board, does that mean that the job is done if we cut corners to get there? Or does it mean we have accumulated even more technical debt? If the point is to publish the package as quickly as possible because it’s a blocker, then we should at least create an issue for writing tests and move it to the top of the todo lane. We need to uphold a certain level of quality in EDS, having a good test coverage and writing good documentation is part of that.

I think the idea was more to add remaining tests in a separate issue. Added an issue for this: #1868.

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Review by vnys — APPROVED** [review]
Linting is missing, apart from that it LGTM 👍🏻

**Pull request merged**